developer-portal/logo.md

```diff
@@ -1,17 +1,37 @@
 # Change your logo
 
-## Add your logo file
+## Add your logo
 
-Add your logo file.
-If you want to overwrite the existing logo, it's at `images/logo.png`.
+To display your logo in the navbar, overwrite the existing image at `./images/logo.png`.
 
-## Adjust your siteConfig
+## Adjust siteConfig
+
+The `siteConfig.yaml` file defines the path to the logo.
 
-The `siteConfig.yaml` has a definition of the path to the header icon.
 ```yaml
 logo: ./images/logo.png
 ```
 
-## Adjust favicon.png
+You must adjust this setting if your custom logo has a different file name or path.
+
+## Adjust logo size
+
+To change the logo image size, you must add the `logo` object to  `./theme.ts`:
+
+```ts
+logo: {
+  height: `50px`,
+  margin: `10px 0`,
+  maxHeight: `100%`,
+  maxWidth: `120%`,
+}, 
+```
+
+The `logo` object controls how the logo image scales within the navbar.
+Note that modifying these options may result in distortion of the logo image.
+Changing the logo image height also affects the total height of the navbar.
+You can read more about [theme customization](https://redoc.ly/docs/developer-portal/theme/) in the documentation.
+
+## Adjust favicon
```

Comment:
You asked a good question about the favicon which, to me at least, indicates it's not super clear what favicons are and how they work in the portal. :). It might be worth adding that info here - just briefly explain where the favicon is displayed and include that it must be a PNG as @adamaltman mentioned.

Reply:
This might be a good place to add a screenshot (:framed_picture: is worth a 1000 words)